pkgs/top-level/python-packages.nix

```diff
@@ -9368,6 +9368,8 @@ in {
 
   semantic-version = callPackage ../development/python-modules/semantic-version { };
 
+  semgrep = callPackage ../development/python-modules/semgrep { };
```

Comment:
If it is just a cli tool we do not want an entry in pythonPackages to be free to overwrite dependency packages.